.github/workflows/main.yml

```diff
@@ -1,22 +1,6 @@
 name: QA
 on: [pull_request]
 jobs:
-  lint:
```

Comment:
If we leave prettier, it should stay here. If that won't enforced by CI, the next person changing the file will get additional formatting diff.

Reply:
I'd like to leave it like Next.js does on default. Hence removing lint rules. I will check how prettier is used in Next.js boilerplate

Reply:
@krzysztofwolski I see that prettier is installed. Next will automatically run eslint (its part of their build). So do you want to have prettier to check on CICD and *fail*?
It will not solve diff problems, because if IDE is not configured locally, changes will be committed and diff will be there. Just CICD will fail later

Maybe I will add `.eslintrc` default for next + add plugin-prettier?

Reply:
(I added)

Reply:
> So do you want to have prettier to check on CICD and fail?

Yes

> It will not solve diff problems, because if IDE is not configured locally, changes will be committed and diff will be there. Just CICD will fail later

Thats the point. It's a submitter's responsibility to push code valid with styleguide. Before the merge submitter have to fix the problems. 

If it's not enforced by the CI, then the next person working on the file with the configured editor will have additional unrelated changes.